### DISCUSSION TITLE
Configuration Location

### DISCUSSION THREAD

**Discussion by CosmicCraftsman** [shadowmoose/RedditDownloader#199] [Q&A] [answered]
Let me start with saying how amazing this app is. I have been experimenting with different configurations within Ubuntu and I screwed up disabled the browser without thinking. I was trying to use it via SSH. I removed the whole folder and unzipped a fresh source folder because I could not find a config file, but the settings remained the same after running the fresh one. Is there a configuration file somewhere I can delete to start over? I'm not the brightest crayon in the box and I just cannot find it.

**Comment by shadowmoose** [comment, marked as answer]
Hey, yeah there is a local data storage for things RMD needs to globally cache. On Linux it should be found at `~/.local/share/RMD`, or `~/.local/share/RedditDownloader`.

**Reply by CosmicCraftsman** [reply]
Awesome, got everything back working again. Thanks!
